src/main/java/banking/CardValidator.java

```diff
@@ -0,0 +1,11 @@
+package banking;
+
+public class CardValidator {
+
+    public boolean isValidCard(CardGenerator generator, String card) {
+        if (card.length() < 16) return false;
+        char checksum = generator.getChecksumFor(card);
+        if (card.charAt(15) != checksum) return false;
+        return true;
```

Comment:
Please simplify this block of code. Please use the IntelliJ Idea hint.

Reply:
fixed